permutation.py

```diff
@@ -1,73 +1,37 @@
-# -*- coding: utf-8 -*-
-''' Given two strings, write a method to decide if one is a permutation of the other.
-Example:
-	
-	Listen, Silent ⇒ True
-	Triangle, Integral ⇒ True
-	Apple, Pabble ⇒ False 
-	
-	First step is to determine the special cases
-		1) s1 or s2 is null ⇒ False
-		2) s1 and s2 are empty strings ⇒ True
-		3) Length s1 != s2 ⇒ False
-	
-	Second step is to see if it is case sensitive
-
-	In the first example :
-		Listen  & Silent ⇒ True  that means that is not case sensitive but for computer 's' ! = 'S'
-
-	So to eliminate this cases we should transform both strings to lowercase
-
-	The first way to solve this problem is to sort this arrays and see if those arrays are equal
-
-	Complexity :
-
-		Time: O(n log n)
-		Space : O (n)
-
-	The second way to solve it's looping through all the letters of each string and count the frequency of each letter(HashMap)
-	Key a : Value 1
-	To do that first we have to transform both string to lowercase
-	 Silent -> silent
-	 Listen -> listen
-
-	 s1) e i l n s t
-	     1 1 1 1 1 1
-	 s2) e i l n s t
-	     1 1 1 1 1 1	
- 	Then we can check if the arrays with frequncy are equal
-
- 	Complexity: 
- 	Time : O(n)
- 	Space : O(n)
-'''
-
-# 
+from collections import Counter
+import numpy as np
```

Comment:
It is OK but in a larger code base it would be a bit suspicious to take on a dependency on numpy for this use case.

Is there a reason to prefer numpy over native python code in this case (asking because I'm curious)?